crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -175,9 +162,9 @@ struct BloomNode {
 }
 
 impl BloomNode {
-    const IN_VIEW: &'static str = "view";
+    pub const IN_VIEW: &'static str = "view";
```

Comment:
if this is going to be public and reused, should it have a more unique name?

Reply:
I'm not sure if this is in the scope of this PR, currently other core nodes all have the same input slot name.

Reply:
I think it makes sense that the input slots have the same name if they serve the same purpose for their respective nodes.